canonical/canonical.com
# WD-17884: Remote Work copy update

## Done

- Minor copy update to the Remote Work page per [copydoc](https://docs.google.com/document/d/1391OL7xQec1SY9HFkv80DFwACbdndCIk-N_sFzAXkPU/edit?tab=t.0). UK English was changed to US English and some additional language explaining the sprints was added.
- Updated the pins on the "recent sprint locations" map to reflect recent sprints. 

## QA

- Open [remote work page](https://canonical-com-1475.demos.haus/careers/company-culture/remote-work)
- See that the map pins have been updated to only the following:
  - Montreal
  - Toronto
  - Lyon
  - Cape Town
  - Frankfurt
  - Prague
  - Copenhagen
  - London
  - Riga
  - The Hague
  - Madrid
  - Vancouver
- Verify the copy doc updates have been accurately reflected on the live page

## Issue / Card

Fixes [WD-17884](https://warthogs.atlassian.net/browse/WD-17884)

## Screenshots

![Screenshot 2024-12-20 at 08 49 25](https://github.com/user-attachments/assets/717e3259-f680-46da-a862-1318df4cbe88)


[WD-17884]: https://warthogs.atlassian.net/browse/WD-17884?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
